main.js

```diff
@@ -0,0 +1,25 @@
+// document.querySelector('.menu-bars').addEventListener('click', () => {
+//   document.querySelector('#mobile-nav').classList.toggle('invisible');
+// });
+
+// document.querySelector('#close').addEventListener('click', () => {
+//   document.querySelector('#mobile-nav').classList.toggle('invisible');
+// });
```

Comment:
- [ ] Kindly remove the `commented` code from your file. `comments` in the code are meant for adding some meaningful `messages` which will help any other developer to understand your code and also help you to easily `maintain` the code in future.
- Please follow `YAGNI` rule according to which always remove the unnecessary code which you wouldn't need.